app/services/github/contributors_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Github
+  class ContributorsService
+    attr_reader :organization
+    attr_reader :members
+
+    def initialize(organization)
+      @organization = organization
+      @members = []
+    end
+
+    def perform
+      first_cursor = Github::Organization.first_member_cursor(@organization)
+      fetch_members first_cursor
+      Users::SyncrhonizeUsersService.new(@members).perform
+      Users::SyncrhonizeRemovedUsersService.new(@members).perform
+    end
+
+    private
+    def fetch_members start_cursor
+      organization_members = Github::Organization.members(@organization, start_cursor)
+      organization_members.nodes.each do |member|
```

Comment:
isn't this the same as ?:
```ruby
@members = organization_members.nodes
```
or is `organization_members.nodes` a different type of object ?, has it its own type ?, if so, does it have a `to_a` method ?, if so you could use it like `@members = organization_members.nodes.to_a`.

If not you could also try?:
```ruby
@members = organization_members.nodes.collect { |member| member }
```

Reply:
@fanta I had to do that in that way, I need to push the `members` elements inside the `organization_members` because the information that Github is returning is paginated.
I tried using the options you mentioned above and that didn't work because if I use `collect` or `to_a` I'm going to be re-creating the `@members` array for every time the endpoint is called, and I need to append those elements.